docs/deploy-from-precompiled-binaries-windows.md

```diff
@@ -0,0 +1,105 @@
+
+* You will find **VirtoCommerce.Platform.3.x.x.zip** file. In this file the site has already been built and can be run without additional compilation. It does not includes all the source code.
+
+* Unpack this zip to a local directory **C:\vc-platform-3**. After that you will have the directory with Platform precompiled files.
+
+## Setup
+
+### Configure application strings
+
+* Open the **appsettings.json** file in a text editor.
+* In the **ConnectionStrings** section change **VirtoCommerce** node (provided user should have permission to create new database):
+
+```json
+    "ConnectionStrings": {
+        "VirtoCommerce" : "Data Source={SQL Server URL};Initial Catalog={Database name};Persist Security Info=True;User ID={User name};Password={User password};MultipleActiveResultSets=True;Connect Timeout=30"
+    },
+
+```
+
+* In the **Assets** section set public url for assets `Assets:FileSystem:PublicUrl` with url of your application, this step is needed in order for display images
+
+```json
+"Assets": {
+        "Provider": "FileSystem",
+        "FileSystem": {
+            "RootPath": "~/assets",
+            "PublicUrl": "https://localhost:5001/assets/" <-- Set your platform application url with port localhost:5001
+        },
+    },
+```
+
+### Running the Platform by CLI "dotnet"
+
+* Install and trust HTTPS certificate
+
+To trust the .NET Core SDK HTTPS development certificate perform run:
+
+```console
+dotnet.exe dev-certs https --trust
+```
+
+Read more about [enforcing HTTPS in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-3.0&tabs=visual-studio#trust)
+
+* Run the Platform by follow command
+
+```console
+dotnet.exe C:\vc-platform-3\VirtoCommerce.Platform.Web.dll
+```
+
+The output in the console will say something like:
+
+```console
+Now listening on: http://localhost:5000
+Now listening on: https://localhost:5001
+```
+
+### First sign in
+
+* Open in your browser follow url `https://localhost:5001` in the warning for not private connections that appears click advanced and continue work. How to remove this error and use a trusted self-signed certificate please read in this article [Trust the ASP.NET Core HTTPS development certificate](https://www.hanselman.com/blog/DevelopingLocallyWithASPNETCoreUnderHTTPSSSLAndSelfSignedCerts.aspx)
+* On the first request the application will create and initialize database. After that you should see the sign in page. Use the following credentials:
+  * Login: **admin**
+  * Password: **store**
+
+### Host on Windows with IIS
+
+VirtoCommerce.Platform project already include the **web.config** file with all necessary settings for running in IIS.
+How to configure IIS application to host ASP.NET Core site please learn more in the official Microsoft ASP.NET Core documentation
+[Host ASP.NET Core on Windows with IIS](https://docs.microsoft.com/en-us/aspnet/core/publishing/iis)
```

Comment:
**web.config** file is already included in the release package. It contains all the necessary settings for running in IIS.
Read more in the official Microsoft ASP.NET Core documentation: [Host ASP.NET Core on Windows with IIS](https://docs.microsoft.com/en-us/aspnet/core/publishing/iis).

Reply:
Fixed